modules/core/src/main/java/org/apache/synapse/mediators/transform/PayloadFactoryMediator.java

```diff
@@ -132,7 +133,9 @@ private boolean mediate(MessageContext synCtx, String format) {
         }
         org.apache.axis2.context.MessageContext axis2MessageContext = ((Axis2MessageContext) synCtx).getAxis2MessageContext();
         StringBuffer result = new StringBuffer();
```

Comment:
StringBuilder should be used instead of StringBuffer, if multiple threads are not using it at the same time.